config/sys.config.src

```diff
@@ -1,6 +1,10 @@
 [
     {hpr, [
-        {base_dir, "/var/data"}
+        {config_worker, #{
+            host => "${HPR_CONFIG_WORKER_HOST}",
+            port => ${HPR_CONFIG_WORKER_PORT},
```

Comment:
Does the relx or whatever does the config parsing support the default subst stuff yet?

```
${HPR_CONFIG_WORKER_PORT:-8080}
```

The config worker expects a string sometimes. Was this meant to be a string?

Reply:
it does will add default port tho